thoth/storages/graph/janusgraph.py

```diff
@@ -251,7 +251,8 @@ def get_runtime_environment(self, runtime_environment_name: str,
             )
 
             if not analysis_document_id:
-                raise NotFoundError(f"No entries for runtime environment {runtime_environment_name!r} found")
+                raise NotFoundError(
+                    f"No entries for runtime environment {runtime_environment_name!r} found")
```

Comment:
:disappointed:

Reply:
ja... merged it because its blocking me :/

Reply:
ok, but fix it